### DISCUSSION TITLE
Nintendo Switch Support

### DISCUSSION THREAD

**Discussion by ComputerDev** [axmolengine/axmol#585] [Ideas]
Do you plan to add support for the Nintendo Switch in the future ?

**Comment by halx99** [comment]
No plan yet

**Comment by DelinWorks** [comment]
To cut down the waiting, Adding support for the switch is not possible as this is an open-source repository and is not possible in nature, and adding support for these kinds of platforms would require an NDA (Non-disclosure agreement) which is a commercial license that's needed from Nintendo. your best bet is to pay a commercial company that has the power to port your axys c++ game to Nintendo somehow

**Comment by ComputerDev** [comment]
I understand I got access but give up
